src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDirectiveDescriptor.cs

```diff
@@ -11,23 +11,24 @@ namespace Microsoft.AspNet.Razor.TagHelpers
     public class TagHelperDirectiveDescriptor
     {
         // Internal for testing purposes.
-        internal TagHelperDirectiveDescriptor(string lookupText,
+        internal TagHelperDirectiveDescriptor(string directiveText,
```

Comment:
Sure the end purpose of the tag helper directive may be different but their core concepts are all the same: affect the creation of `TagHelperDescriptor`s  Their core concept is handled by the `TagHelperDescriptorResolver` who only understands provided `TagHelperDirectiveDescriptor`s, no reason to build a separate code path outside of this core `TagHelperDescriptor` building mechanism